### ISSUE TITLE
Processor and RAM blow up upon starting VS Code, application window becomes unresponsive

### ISSUE THREAD

**Issue by AndyLuckert** [microsoft/vscode#99968]
**- VSCode Version:**
1.46.0
a5d1cc28bb5da32ec67e86cc50f84c67cc690321
x64

**- OS Version:**
Operating System: Ubuntu 18.04.4 LTS
Kernel: Linux 5.3.0-59-generic
Architecture: x86-64

**Steps to Reproduce:**

1. Update `VS Code` to latest version `1.46.0`.
2. Start it with all extensions enabled. Watch the processor and RAM blow up, then see the `VS Code` window freeze and being asked every once in a while "The window is no longer responding: do you want to continue waiting for that window, reopen or close it?". After the processor and RAM blew up, both return back to normal, but `VS Code` remains unresponsive.
-> The RAM and processor almost blowing up upon starting the extension has been happening recently on a regular basis, but at least it recovered at some point and I was able to use `VS Code` normally. As of this morning though, the application window becomes unresponsive.

**Further comments:**

1. When starting with all extensions disabled the window doesn't become unresponsive nor the RAM and processor blows up. Command used: `code --disable-extensions`
2. As a reaction to this, I uninstalled several extensions I actually never use. Nevertheless, when starting with all the remaining extensions enabled, the same problem as described above occurs again.

**Additional question:**
Given the assumption were true that the problem were mainly rooted in a specific extension, how can I achieve to isolate WHICH one it is of the 25 extensions installed?
Is there any command line option whatsoever which lists all extensions along with their processor and RAM usage?
Or could it be that the problem is also related to the latest VS Code update?

Does this issue occur when all extensions are disabled?: No
-> If an extension were the cause, how can I find out which one it is?

**Comment by gjsjohnmurray** [comment]
One strategy is to bisect. If you create a workspace (i.e. XXX.code-workspace) first, you can disable half your extensions for this workspace, then test, then either disable half of those that remain active, or re-enable half of those you previously disabled. Using a workspace means you don't also have to re-enable everything at the end. Simply discard XXX.code-workspace.

**Comment by deepak1556** [comment]
@AndyLuckert were you able to narrow down on an extension, that might be causing this issue ? I don't have anything to add to what @gjsjohnmurray mentioned for bisecting the extensions, please give it a try. Thanks!

**Comment by AndyLuckert** [comment]
Thanks for your advice!
I made it work again, but cannot tell you unfortunately which factor was deciding.
Nevertheless, I'll list them in the following for if it helps someone with a similar problem:

1. Uninstall all unnecessary `extensions`.
2. Save your unsaved `workspace` to a real work-space-file (`VS Code` characteristic).
3. Open a new` workspace`, try out without `extensions` and with all you remaining ones.
4. Reopen the actual `workspace` of interest.

Apart from that, I uninstalled `anaconda` completely and replaced it with the `linuxbrew PIP-based` package manager.

Well, now it works like a charm, so all good :)
